package.json

```diff
@@ -37,6 +36,7 @@
     "LICENSE"
   ],
   "main": "./dist/turbopuffer.js",
+  "types": "./dist/turbopuffer.d.ts",
```

Comment:
this types file is different from `types(.ds).ts`. This needs to be `"types": "./dist/types.d.ts",` but we'll need to change the filename to .ts instead of .d.ts (like you did); tsconfig's `"declaration": true,` automatically handles the .d.ts creation for us

Reply:
I'm not sure I follow! AFAIU, the `types` config here should point to the declaration file for the main module's types, which imports the declaration files for all submodules.

Reply:
ah yes, was confused there for a min my bad. I'll update the versions and approve and publish. Thank you for making these changes 🙌

Reply:
Of course! Thanks for making the client work in non-node environments! 😍